### DISCUSSION TITLE
Git Add Untracked Changes fails - [MASTER] warn: Cannot create a string longer than 0x1fffffe8 characters

### DISCUSSION THREAD

**Discussion by MattSidney** [requarks/wiki#4707] [Error / Bug Report]
When first setting up git sync and attempting to 'Add Untracked Changes' it fails. Initially it appears to work but eventually spams the warn below and fails to update git remote.  
```
2021-11-16T00:39:56.621Z [MASTER] info: (STORAGE/GIT) Adding all untracked content...
2021-11-16T00:40:06.974Z [MASTER] info: (STORAGE/GIT) Adding page home/redacted.md...
2021-11-16T00:40:07.402Z [MASTER] info: (STORAGE/GIT) Adding page home/redacted.md...
2021-11-16T00:40:45.579Z [MASTER] warn: Cannot create a string longer than 0x1fffffe8 characters
...
2021-11-16T00:40:46.342Z [MASTER] warn: Cannot create a string longer than 0x1fffffe8 characters
```

Not sure if there is a a way to enable additional logging to better understand what is going wrong here.

**Comment by MattSidney** [comment]
still seeing this. How is git sync currently setup? hoping to gather some more information on this if I can. 
Seems like the upload is running into a size issue? https://stackoverflow.com/questions/68230031/cannot-create-a-string-longer-than-0x1fffffe8-characters-in-json-parse

Is there a way to get more detailed logs from this sync process? @NGPixel

**Comment by Asgoret** [comment]
Same here. @NGPixel, but in my case I`m trying to create backup to S3 Generic. We have data about 5Gb (if it necessary)

**Reply by NGPixel** [reply]
This error is completely unrelated to S3. Make sure you don't have Git enabled as well, which is where the error would be coming from...

**Reply by Asgoret** [reply]
Only S3 Generic. Ill just ended move from confluence via local storage import. And try to create backup to S3. When there was several files all works perfect, but when migration ended starting to get this error on DB dump to S3. Try in 2.5.277|8|9 versions.

**Reply by Asgoret** [reply]
And when try to create manual backup to local strorage, gz archive created with 87 bytes weight. When try to CP it from K8S pod got error: `tar: removing leading '/' from member names` (dunno is that familiar to this issue)

**Reply by Asgoret** [reply]
Dump on local storage also get this warning.

**Reply by NGPixel** [reply]
Thanks for reporting.

The error is unfortunately not very useful for locating the root cause. If you import the page mentioned just before the error in an empty wiki.js installation, and try to export it back, do you still get the same error?

**Reply by Asgoret** [reply]
It would hard to find exact page... I tried backup on <100 pages, now there are 763...Ill try to find problem page(s) after 10th may) Is there any suggestions what I can search in raw html code? Some page have a lot of attachments (design department for example), with video and so on

**Comment by MattSidney** [comment]
still seeing this issue. 
closest thing I could find when looking for this issue originally was https://stackoverflow.com/questions/68230031/cannot-create-a-string-longer-than-0x1fffffe8-characters-in-json-parse
Is that potentially relevant to the issue I am seeing? 

also found this from another project: https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/423

**Reply by 7eptem** [reply]
same problem here, i am getting Cannot create a string longer 0x1fffffe8 characters

**Reply by 7eptem** [reply]
@MattSidney  have you find any fix ? because i think i will move from wikiJS sice i see that this problem is from 2021
